roofit/roofitcore/src/RooRealVar.cxx

```diff
@@ -91,7 +91,7 @@ RooRealVarSharedProperties& RooRealVar::_nullProp()
 ////////////////////////////////////////////////////////////////////////////////
 /// Default constructor.
 
-RooRealVar::RooRealVar()  :  _error(0), _asymErrLo(0), _asymErrHi(0), _binning(new RooUniformBinning())
+RooRealVar::RooRealVar()
```

Comment:
The major challenge is whether the relevant code handles this 'new' state (i.e. `_binning` can be null if the user uses the default constructor directly or indirectly (creation done by I/O but reading not 'yet' done).

A cursory look indicates a few places where there might be a problem.

Reply:
Yes you are right, and the failing unit test suggests that someone was expecting the default constructor to work outside of IO, and hence a test was added. Need to think more about it